src/app/public/modules/grid/grid-column.component.ts

```diff
@@ -27,6 +27,13 @@ export class SkyGridColumnComponent implements OnChanges {
   @Input()
   public heading: string;
 
+  /**
+   * Specifies a template to display inside an inline help popup for this column.
+   * This property accepts [an Angular TemplateRef](https://angular.io/api/core/TemplateRef) class.
+   */
+  @Input()
+  public inlineHelpPopover: TemplateRef<any>;
```

Comment:
Can it be the following, to be more specific?
```suggestion
  public inlineHelpPopover: TemplateRef<SkyPopoverComponent>;
```

Reply:
Yes, changed.